index.html

```diff
@@ -356,7 +356,7 @@ <h1 class="title">Promise.prototype.finally</h1>
 		</emu-alg>
 	</emu-clause>
 
-	<!-- es6num="25.4.1.8" -->
+	<!-- es6num="25.4.4.5" -->
```

Comment:
It's 25.4.4.5 in the spec now, so presumably this change is fine. Not sure where 25.4.1.8 came from. It should remain unchanged from ES6.